ViAn/GUI/mainwindow.cpp

```diff
@@ -2,13 +2,15 @@
 #include "ui_mainwindow.h"
 #include <QMessageBox>
 #include <iostream>
+#include <sstream>
 #include <QCloseEvent>
 #include <QColorDialog>
 #include <chrono>
 #include <thread>
 #include "icononbuttonhandler.h"
 #include "inputwindow.h"
 #include "Video/shapes/shape.h"
+#include <qdebug.h>
```

Comment:
Is `#include <qdebug.h>` needed? (I tried commenting it out and it was no prolem.)